### ISSUE TITLE
Word Detection Error?

### ISSUE THREAD

**Issue by JuanSlaterT** [7coil/DiscordForOffice#14]
Sometimes when i'm supposed to be working in word, in discord only appears this

![image](https://user-images.githubusercontent.com/20392025/95237356-b4ec3080-07cd-11eb-91fa-f85c75cf4c7e.png)

Not sure why is happening, but just checked this when I was making my word document and I had back to check my discord game status

**Comment by 7coil** [comment]
This is an intended feature. If you click off Microsoft Word, this "Not Active" screen appears.

This is because the event hook that I'm using triggers when a user clicks off _and_ when a user closes a file.
Without this, there is no other way of tracking (that I know of at the moment) when a user closes a file.

This means that clicking off a file will display this message. Sorry!

_Without this, closing a file would keep the old information up, which I feel is less accurate than this current compromise._

**Comment by 7coil** [comment]
If there is actually a new way of detecting when a file is closed, that would be very helpful for me to stop this issue.

**Comment by JuanSlaterT** [comment]
> This is an intended feature. If you click off Microsoft Word, this "Not Active" screen appears.
> 
> This is because the event hook that I'm using triggers when a user clicks off _and_ when a user closes a file.
> Without this, there is no other way of tracking (that I know of at the moment) when a user closes a file.
> 
> This means that clicking off a file will display this message. Sorry!
> 
> _Without this, closing a file would keep the old information up, which I feel is less accurate than this current compromise._

I dont think this as "intended" just tested letting my game status "opened" in another monitor and just started to write some words in my word documment, but it still appearing as "Not Active" I made some clicks too but nothing

I closed the file and re opened it again and it appeared but after a few minutes it went back to Not Active for some reason.

**Comment by 7coil** [comment]
Now that's an issue.
What version are you using?

![image](https://user-images.githubusercontent.com/13886925/95453621-1560a680-0963-11eb-9266-86b03da5e256.png)

**Comment by JuanSlaterT** [comment]
![image](https://user-images.githubusercontent.com/20392025/95536487-c6416280-09b0-11eb-9c8a-41909efc0ae6.png)
it say latest

**Comment by 7coil** [comment]
This issue has gone stale... even if it was my fault for not replying.

If the issue is still there, reply and I'll get back to you.

**Comment by 7coil** [comment]
Good news - There actually _is_ a way to detect this:

https://social.msdn.microsoft.com/Forums/vstudio/en-US/f0f447c5-0d0d-40fb-9fa2-fe14b2eb129d/how-to-detect-a-word-document-is-being-closed-from-a-vsto-addin?forum=vsto

**Comment by 7coil** [comment]
Unfortunately it doesn't seem to work for when the _last_ document is closed for some reason...

**Comment by garridinsi** [comment]
Hi, I don't know if has to do with this, but when im editing a file it doesn't appear on Discord, it gets stuck on "No file open, Welcome Screen" as on the image

![imagen](https://user-images.githubusercontent.com/53048643/109865214-b5450680-7c5b-11eb-9882-3e80f288fbc0.png)

**Comment by 7coil** [comment]
Yeah, I think it's the same issue still.
